payment-rfcs/ewc-rfc008-payment-data-confirmation.md

```diff
@@ -200,36 +201,36 @@ The authorisation response, as defined in EWC RFC 002 [3], is extended to includ
 The holder wallet must display all attributes (keys and values) contained in the <strong><code>payment_data</code></strong> object to the user as part of releasing the PWA credential and obtaining an explicit user action, signalling they confirm the details presented. It may localise (translate) keys, localise (adapt) number formats and replace or amend currency codes with symbols where appropriate and unambiguous. 
 
 ## 5.2 Verifier
+To verify the VP token received in the presentation response, the verifier needs to have the following information.
+This information may have originated from themselves, or been relayed to them by the intermediary in a three-party model:
 
-The integrity and authenticity of the payment data can be verified by the party who knows the public key of the key pair whose possession was proven by the user when the Payment Wallet Attestation was issued as per EWC RFC 007 [5].
+* The base64url encoded <strong><code>transaction_data</code></strong> string that was sent to the wallet as part of the authorization request.
+* In a three party model - The original input from the user on the payment details, i.e. payee, amount, currency at least, plus any optional parameters. Obtained from the intermediary through industry-specific protocols like 3DS or open banking APIs.
 
-The verifier further needs to have the following information, either directly from the wallet or relayed by the intermediary in a three-party model:
+With this information at hand, plus the VP token presented by the wallet, the verifier performs the following:
 
-* The PWA issued into the user’s wallet and corresponding to the funding source (card or account) in question was presented from the user’s wallet and signed by the user.
-* The <strong><code>transaction_data</code></strong> object was constructed and sent to the wallet as part of the authorization request.
-* The key-binding JWT incl. the hash of the <strong><code>transaction_data</code></strong> object.
-* The original input from the user on the payment details, i.e. payee, amount, currency at least, plus any optional parameters. In a two-party model, the user typically inputs this data into a form the bank presents in their online or mobile banking. In a three-party model, the intermediary relays this data, which he has agreed with the user, through industry-specific protocols like 3DS or open banking APIs.
+1. Validate the VP Token as described in OpenID4VP [4] for the IETF SD-JWT VC credential format.
+  * This should include validating the key binding JWT, as a PWA is always bound to a key.
```

Comment:
Does this validation include ensuring that the PWA is presented by the same user as it was issued (and bound) to?

Reply:
short answer: yes

longer answer: 
validating the key binding jwt ensures that the PWA was presented by someone that controls the private key that was bound to the PWA when it was issued. And that private key we know to be device bound (certified wallet solution provider should ensure this). so you have PWA -> key -> device. and then the device being bound to user is generally assumed / an accepted risk.